src/components/pages/maintenance/maintenance.scss

```diff
@@ -78,7 +78,8 @@
 
         &.active {
           color: themed('colorContentText');
-          border-bottom: 4px solid themed('colorHeaderBorderColor');
+          border-bottom: 4px solid;
+          border-bottom-color:  #FFFFFF;
```

Comment:
This isn't going to work. All colors need to come from the theme file. If you go to settings and change the page to the light theme, the border will not be visible anymore. Notice how in the previous code the color was defined using the `themed` mixin. The only change you need is to change the value `colorHeaderBorderColor` maps to in the theme file.